src/platform/ios.rs

```diff
@@ -98,6 +98,8 @@ pub trait WindowExtIOS {
   /// and then calls
   /// [`-[UIViewController setNeedsStatusBarAppearanceUpdate]`](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621354-setneedsstatusbarappearanceupdat?language=objc).
   fn set_prefers_status_bar_hidden(&self, hidden: bool);
+
+  fn set_badge_count(&self, count: i32);
```

Comment:
we need to addd `EvnetLoopWindowTargetExtIOS` and add a method similar to this on it

Reply:
We need to add a trait, right?